### DISCUSSION TITLE
New to Plus

### DISCUSSION THREAD

**Discussion by SerenaIntusurg** [PlusToolkit/PlusLib#847] [General]
Hi,
I'm looking to get an ultrasound probe and start using plus.  I'm looking at the Interson GP-01, but  I don't see a device set available for it in the fCal program.  How do set up that file correctly?  Is there a preferred ultrasound probe to use with this program?

Thanks,
Serena

**Comment by ungi** [comment]
At the moment, new Interson probes are not supported by PLUS yet. We plan to update the support soon, I just wanted to make sure you know about this. I would not use fCal, unless you must provide a sterile intraoperative way to calibrate your probe. The only advantage of fCal is that it can be designed to work in a sterile box. However, if you can mount the tracker on the probe in a reproducible way, it's better to calibrate only once with a tracked stylus and use that calibration with all your scans. Here is a tutorial how to calibrate with a tracked stylus: http://www.slicerigt.org/wp/user-tutorial/ (see U-31). You can change the ultrasound device part in any example PLUS config file. It should not matter for the rest of the config file.

**Comment by SerenaIntusurg** [comment]
So which ultrasound system would you recommend I get to work with Plus?

**Reply by ungi** [reply]
I personally use Telemed ultrasound machines. They are very reliable and the programming interface has not changed in a  long time, so it's easy to work with them. PLUS also supports video frame grabbers, so you could use a high-end ultrasound with a standard video output (DVI, RGB, etc.), and capture the frames with an Epiphan frame grabber. Clarius wireless ultrasounds will also be supported soon. They are quite special because of the wireless design, but they have very good image quality at a relatively low cost.

**Comment by SerenaIntusurg** [comment]
Are there specific epiphan grabbers that are supported or not supported by Plus?  I was looking at a pretty barebones one: https://www.amazon.com/Epiphan-DVI2USB-Video-Grabber-ESP1137/dp/B071ZZPCCW/ref=sr_1_26?dchild=1&keywords=epiphan+recording&qid=1633413619&sr=8-26

**Reply by ungi** [reply]
That looks like the one I was using too. This is the PLUS user guide page to set it up: http://perk-software.cs.queensu.ca/plus/doc/nightly/user/DeviceEpiphan.html

**Comment by SerenaIntusurg** [comment]
Ok – I can start the server, but it looks like I nee the OpenIGTLinkIF to get the data into Slicer.  Doe sthis come as a prebuilt binary or is the only way for me to pull the repository and compile it?

From: Tamas Ungi ***@***.***>
Sent: Tuesday, October 5, 2021 4:59 AM
To: PlusToolkit/PlusLib ***@***.***>
Cc: Serena Wong ***@***.***>; Author ***@***.***>
Subject: [EXTERNAL] Re: [PlusToolkit/PlusLib] New to Plus (Discussion #847)


That looks like the one I was using too. This is the PLUS user guide page to set it up: http://perk-software.cs.queensu.ca/plus/doc/nightly/user/DeviceEpiphan.html<https://urldefense.com/v3/__http:/perk-software.cs.queensu.ca/plus/doc/nightly/user/DeviceEpiphan.html__;!!PEK3uBjM8x5RqC4b!Tm1Wi6mdTVBG_6_It8oR8z0KZ5LNenzyPYhl6zaFT0mTLVqKWc89BeEWUpzzPn5rvto$>

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://urldefense.com/v3/__https:/github.com/PlusToolkit/PlusLib/discussions/847*discussioncomment-1429736__;Iw!!PEK3uBjM8x5RqC4b!Tm1Wi6mdTVBG_6_It8oR8z0KZ5LNenzyPYhl6zaFT0mTLVqKWc89BeEWUpzzd9vijSk$>, or unsubscribe<https://urldefense.com/v3/__https:/github.com/notifications/unsubscribe-auth/ALLSZYQ4IEAVKJL6W2R24N3UFLR7RANCNFSM5FJWFYAQ__;!!PEK3uBjM8x5RqC4b!Tm1Wi6mdTVBG_6_It8oR8z0KZ5LNenzyPYhl6zaFT0mTLVqKWc89BeEWUpzztSmNdyY$>.
Triage notifications on the go with GitHub Mobile for iOS<https://urldefense.com/v3/__https:/apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675__;!!PEK3uBjM8x5RqC4b!Tm1Wi6mdTVBG_6_It8oR8z0KZ5LNenzyPYhl6zaFT0mTLVqKWc89BeEWUpzzzrb91yA$> or Android<https://urldefense.com/v3/__https:/play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign*3Dnotification-email*26utm_medium*3Demail*26utm_source*3Dgithub__;JSUlJSU!!PEK3uBjM8x5RqC4b!Tm1Wi6mdTVBG_6_It8oR8z0KZ5LNenzyPYhl6zaFT0mTLVqKWc89BeEWUpzzIElNios$>.

________________________________
NOTE THAT THIS EMAIL ORIGINATED FROM OUTSIDE OF INTUITIVE SURGICAL.
Be alert for fraudulent emails that spoof internal ***@***.***" email addresses. Report any suspicious emails using the "Report Phish" button. Click KB0014776<https://intusurg.service-now.com/it_sp?id=int_kb_article&sys_id=181cb0e01b9f44500fa1fe631a4bcb00> for more information on the "Report Phish" button and to learn more about differentiating phishing from spam and bulk email, please review KB0014940<https://intusurg.service-now.com/isp?id=int_kb_article&sys_id=fb3b3a1d1b999c940fa1fe631a4bcbfa>.

**Reply by Sunderlandkyl** [reply]
The OpenIGTLinkIF module comes with the SlicerOpenIGTLink extension. You can install it through the extension manager. https://slicer.readthedocs.io/en/latest/user_guide/extensions_manager.html
